drivers/gpio/gpio_mcux.c

```diff
@@ -321,8 +321,7 @@ static void gpio_mcux_port_isr(const struct device *dev)
 
 	/* Clear the port interrupts */
 	config->port_base->ISFR = int_status;
-#elif (defined(FSL_FEATURE_GPIO_HAS_INTERRUPT_CHANNEL_SELECT) && \
-		FSL_FEATURE_GPIO_HAS_INTERRUPT_CHANNEL_SELECT)
+#else
```

Comment:
Was this feature macro dropped in the SDK? If not, why isn't it defined for this GPIO IP block?